src/sync/installation.ts

```diff
@@ -365,6 +367,15 @@ export const handleBackfillError = async (
 		return;
 	}
 
+	// TODO: replace with "instanceof" when sequelize version is upgraded to some modern one
+	// Capturing errors like SequelizeConnectionError, SequelizeConnectionAcquireTimeoutError etc
+	// that are not exported from sequelize
+	if (String(err.name).toLowerCase().includes("sequelize")) {
```

Comment:
`NOT` your fault, `NOT` this PR, but the way we determine the error type sucks! Really missed the java checked exception.